packages/cowabloga/cowabloga.0.2.2/opam

```diff
@@ -10,7 +10,7 @@ build: [make "all"]
 install: [make "install"]
 remove: ["ocamlfind" "remove" "cowabloga"]
 depends: [
-  "cow" {>= "2.0.0"}
+  "cow" {>= "2.0.0" & < "2.3.0"}
   "omd" {>= "0.8.2"}
   "lwt" {>= "2.4.3"}
   "cohttp" {>= "0.15.0"}
```

Comment:
Given that you are touching these can you also add < 1.0.0 bounds on cohttp dependencies in cowabloga?